scan/src/pages/OracleScriptIndexPage.re

```diff
@@ -21,196 +21,210 @@ module Styles = {
 let make = (~oracleScriptID, ~hashtag: Route.oracle_script_tab_t) => {
   let oracleScriptSub = OracleScriptSub.get(oracleScriptID);
 
-  <Section pbSm=0>
-    <div className=CssHelper.container>
-      <Heading value="Oracle Script" size=Heading.H4 marginBottom=40 marginBottomSm=24 />
-      <Row.Grid marginBottom=40 marginBottomSm=16 alignItems=Row.Center>
-        <Col.Grid col=Col.Eight>
-          <div className={Css.merge([CssHelper.flexBox(), Styles.idCointainer])}>
-            {switch (oracleScriptSub) {
-             | Data({id, name}) =>
-               <>
-                 <TypeID.OracleScript id position=TypeID.Title />
-                 <HSpacing size=Spacing.sm />
-                 <Heading size=Heading.H3 value=name />
-               </>
-             | _ => <LoadingCensorBar width=270 height=15 />
-             }}
-          </div>
-        </Col.Grid>
-        <Col.Grid col=Col.Four>
-          <div className=Styles.infoContainer>
-            <Row.Grid>
-              <Col.Grid col=Col.Six colSm=Col.Six>
-                <div className={CssHelper.flexBox(~direction=`column, ())}>
-                  <Heading value="Requests" size=Heading.H4 marginBottom=8 align=Heading.Center />
-                  {switch (oracleScriptSub) {
-                   | Data({requestCount}) =>
-                     <Text
-                       value={requestCount |> Format.iPretty}
-                       size=Text.Xxl
-                       align=Text.Center
-                       block=true
-                     />
-                   | _ => <LoadingCensorBar width=100 height=15 />
-                   }}
-                </div>
-              </Col.Grid>
-              <Col.Grid col=Col.Six colSm=Col.Six>
-                <div className={CssHelper.flexBox(~direction=`column, ())}>
-                  <div
-                    className={Css.merge([
-                      CssHelper.flexBox(~justify=`center, ()),
-                      Styles.titleSpacing,
-                    ])}>
-                    <Heading value="Response time" size=Heading.H4 align=Heading.Center />
+  switch (oracleScriptSub) {
+  | NoData => <NotFound />
+  | _ =>
+    <Section pbSm=0>
+      <div className=CssHelper.container>
+        <Heading value="Oracle Script" size=Heading.H4 marginBottom=40 marginBottomSm=24 />
```

Comment:
Here as well
@evilpeach What do you think?

Reply:
Here's too. @pzshine